test/integration/krane_test.rb

```diff
@@ -53,8 +55,8 @@ def test_render_black_box
   end
 
   def test_deploy_black_box_success
-    setup_template_dir("hello-cloud") do |target_dir|
-      flags = "-f #{target_dir} --render-erb --bindings deployment_id=1 current_sha=123"
+    setup_template_dir("hello-cloud", subset: %w(bare_replica_set.yml)) do |target_dir|
+      flags = "-f #{target_dir} --bindings deployment_id=1 current_sha=123"
```

Comment:
#632 is removing `bindings / current-sha` flag from deploy and the subset you specify doesn't need them anyway /